### ISSUE TITLE
Draft accounting, billing and payroll workflow

### ISSUE THREAD

**Issue by benhylau** [hyphacoop/organizing#89]
Re-ticketed from https://github.com/hyphacoop/organizing/issues/84#issuecomment-519755417

## Shared Description

:speaking_head: **Loomio:** https://loomio.hypha.coop/d/Y28GOpkF/draft-accounting-billing-and-payroll-workflow  
:date: **Due date:** August 16, 2019  
:dart: **Success criteria:** Have a draft for members and accountant to review  

## To Do

- [x] Financial accounts
- [x] Payment methods
- [x] Client billing
- [x] Payroll
- [x] Tax withholding
- [x] Identify services
  - [x] trial services
    - [x] ~~Odoo~~ #96 deferred one year
      - [x] deploy self-hosted server -- @ASoTNetworks @YurkoWasHere
    - [x] Freshbooks -- @benhylau
    - [x] Wave -- @benhylau 
    - [ ] Quickbooks Online -- @benhylau 
    - [ ] Xero -- ~~@patcon~~ @benhylau 
  - [x] seek advice from accountant -- @benhylau( `blocked` on confirmation and making intro in #70) 

Once drafted, next steps would be:

- Review/approval process
- Identify and document responsibilities, per hyphacoop/handbook#13

**Comment by dcwalk** [comment]
@benhylau -- to be clear what are those above todos? Drafting a process?
We should also include:
- identify and document responsibilities, per https://github.com/hyphacoop/handbook/issues/13
- identify services
- review/approval process

**Comment by benhylau** [comment]
>to be clear what are those above todos? Drafting a process?

Identifying the services and steps, how it all works together, as a draft for review.

---

>identify and document responsibilities, per hyphacoop/handbook#13
>review/approval process

The success criteria is to have the draft for review, so I feel these two may be out of scope? I have added this as next step.

**Comment by patcon** [comment]
Is this issue blocked on reviewing/choosing software? Or does it encompass software selection?
Re: https://loomio.hypha.coop/d/eaH0FZNR/tool-talk-financial-accounting-software

I ask because I was going to create a ticket for gathering details on what we're demo'ing and talking about, but I'm unclear whether you're seeing that as part of this, or part of a separate infra task.

**Comment by patcon** [comment]
Maybe we could break this up? This feels kinda giant and bigger than one person, or at least could be divided much smaller.

Per-platform tickets like the Odoo one maybe seem too granular, but per-category research and testing might make sense. Thoughts?

**Comment by patcon** [comment]
On call, @benhylau and I sorted confusion on ownership of accounting software selection: It's part of this ticket, and @benhylau is taking it (not part of infra wg). I've added clarity in OP on which platforms we've tested or will test.

**Comment by patcon** [comment]
As per conversation at last week's all-hands, we've postponed consideration of an open-source financial accounting system for one year. Tracked here: https://github.com/hyphacoop/organizing/issues/107.

**Comment by benhylau** [comment]
Finally finished writing up the draft for [Financial Accounts and Practices](https://hackmd.io/aEw_aVVIQwSWRxtKQa7wuQ?view).

**Comment by benhylau** [comment]
We should be able to close this once https://github.com/hyphacoop/handbook/pull/22 is merged.

**Comment by benhylau** [comment]
Closing this since:

>🎯 Success criteria: Have a draft for members and accountant to review

is fulfilled and not closing PR for now https://github.com/hyphacoop/handbook/pull/22#issuecomment-548466808
